src/officehours_ui/static/css/main.css

```diff
@@ -687,3 +687,16 @@ div[role=main] {
 .badge-success {
 	background-color: var(--custom-green);
 }
+
+a.nav-link:hover {
+	background-color: #004881;
+	border-radius: 3px;
+}
+
+a.dropdown-item:hover {
+	background-color: #C0C0C0 !important;
+}
+
+input.dropdown-item:hover {
+	background-color: #C0C0C0 !important;
+}
```

Comment:
@mfldavidson or @teshsopher, we may need some input here. @vjcao also added a gray hover background, which was out of scope for the issue, but a logical addition. Do we like this? Do we need to tweak the color for adequate contrast/consistency?
<img width="644" alt="Screen Shot 2021-05-05 at 12 26 13 PM" src="https://user-images.githubusercontent.com/35741256/117176034-75a2b400-ad9d-11eb-814d-a48361eaef57.png">

Reply:
@ssciolla I think it looks good, and the color contrast is great too! 

My only question is: does this menu show up when the user clicks the "hamburger" menu in the top right? If it is, then I would suggest having the hamburger menu appear as an "X" or some other "close menu" icon while the menu is open so that users can more easily see that they can close the menu.

Reply:
Thanks for the input on highlight colors and hamburger-menu suggestion, @teshsopher. Note that this picture is the mobile (small viewport) appearance; there is no hamburger menu with a large viewport. While related to these changes, I feel this is a separate issue that would require additional work and research. Would you mind taking a moment to document this as a new issue? If you're pretty busy, I can, just let me know.

Reply:
I can do that!

Reply:
Thank you!